x/fixationstore/README.md

```diff
@@ -0,0 +1,67 @@
+# `x/fixationstore`
+
+## Abstract
+
+This document specifies the `fixationstore` module of Lava Protocol.
+
+This module primarily serves as a utility for other modules in the Lava Protocol. It's not designed for direct user interaction, except for a limited set of queries intended for debugging. As such, it functions as an essential support utility within the protocol's ecosystem.
+
+The fixationstore allows other modules to create fixation stores that manages lists of entries with versions in the store. Fixation entries may change over time, and their versions must be retained on-chain as long as they are referenced. For examples, an older version of a plan is needed as long as the subscription that uses it lives.
+
+## Contents
+
+- [Concepts](#concepts)
+  - [Entry](#entry)
+    - [Reference Count](#reference-count)
+    - [Stale Period](#stale-period)
+  - [Usage](#usage)
+- [Parameters](#parameters)
+- [Queries](#queries)
+- [Transactions](#transactions)
+- [Proposals](#proposals)
+
+## Concepts
+
+### Entry
+
+A fixated entry version is identified by its index (name) and block (version). The "latest" entry version is the one with the highest block that is not greater than the current (ctx) block height. If an entry version is in the future (with respect to current block height), then it will become the new latest entry when its block is reached.
```

Comment:
```suggestion
A fixated entry version is identified by its index (name) and block (version). The "latest" entry version is the one with the highest block that is not greater than the current (which is retrieved from the context) block height. If an entry version is in the future (with respect to current block height), then it will become the new latest entry when its block is reached.
```

Reply:
done. see https://github.com/lavanet/lava/commit/233d8bb0ba944648d6d119ca62fd3fc81de9b301